irohad/network/impl/peer_communication_service_impl.cpp

```diff
@@ -15,6 +15,7 @@ limitations under the License.
 */
 
 #include "network/impl/peer_communication_service_impl.hpp"
+#include "backend/protobuf/from_old_model.hpp"
```

Comment:
Please use absolute path (missed shared_model)

Reply:
We can't :(